app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -210,7 +211,8 @@ class Core(
                 PromptMiddleware(),
                 AdsTelemetryMiddleware(adsTelemetry),
                 LastMediaAccessMiddleware(),
-                HistoryMetadataMiddleware(historyMetadataService)
+                HistoryMetadataMiddleware(historyMetadataService),
+                SearchTermTabGroupMiddleware()
```

Comment:
Bit of a nitpicky drive-by, but wanted to call out that we could avoid diffs like this in the future by using [trailing commas](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas) everywhere, now that they're available. I've been trying to get in the habit, personally. Just something to keep in mind for the future, no action needed here

Reply:
I'm not sure.  Are we doing trailing commas?  I've been making sure to remove them when I see them.  I lean towards no trailing commas for readability.  But I think we should decide if this is something we want to do in the future.

Reply:
Drive by comment: I'm for standardizing trailing commas to keep the diffs just a bit cleaner and the history easier to follow since in many cases it acts as documentation for the feature.

Reply:
@Mugurell Yes, we've got an agreement as a team we'll now add the trailing commas in all future changes.  Thanks!